src/Neuron/Zettelkasten/Markdown/Extension.hs

```diff
@@ -0,0 +1,48 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Neuron.Zettelkasten.Markdown.Extension
+  ( setTableClass,
+  )
+where
+
+import qualified Data.List.NonEmpty as NE
```

Comment:
[Q] I did not find a Relude equivalent to `NE.zip` so I kept this one

Reply:
That's fine, yea.